src/main/java/mate/academy/spring/service/impl/UserServiceImpl.java

```diff
@@ -4,24 +4,24 @@
 import mate.academy.spring.dao.UserDao;
 import mate.academy.spring.model.User;
 import mate.academy.spring.service.UserService;
-import mate.academy.spring.util.HashUtil;
+import org.springframework.security.crypto.password.PasswordEncoder;
 import org.springframework.stereotype.Service;
 
 @Service
 public class UserServiceImpl implements UserService {
     private static final int SALT_LENGTH = 10;
```

Comment:
You need to remove this

Reply:
agree, it's a bad practice to leave unused code